setup.py

```diff
@@ -2,7 +2,7 @@
 
 
 setup(name='rocket-python',
-      version='1.2.14',
+      version='1.2.15',
```

Comment:
Please do not touch version numbers, I update it automatically wen I'm doing releases.
```suggestion
      version='1.2.14',
```